src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/email/ReportSenderTest.java

```diff
@@ -39,23 +35,27 @@
     @Test
     public void should_send_email_to_all_recipients() throws Exception {
         // given
-        String[] recipients = Arrays.array(
-            "foo@hmcts.net",
-            "bar@hmcts.net"
-        );
+        String reportRecipients = "Foo <foo@hmcts.net>, bar@hmcts.net";
 
         ReportsService reportsService = mock(ReportsService.class);
 
         greenMail.setUser(TEST_LOGIN, TEST_PASSWORD);
-        ReportSender reportSender = new ReportSender(getMailSender(), reportsService, recipients);
+        ReportSender reportSender = new ReportSender(getMailSender(), reportsService, reportRecipients);
 
         // when
         reportSender.send();
 
         // then
         MimeMessageParser msg = new MimeMessageParser(greenMail.getReceivedMessages()[0]).parse();
 
-        assertThat(msg.getTo()).extracting(Address::toString).containsExactlyElementsOf(asList(recipients));
+        assertThat(msg.getTo())
+            .extracting(Address::toString)
+            .hasSize(2)
+            .containsExactlyElementsOf(
```

Comment:
why not check contains exactly only `Foo <foo@hmcts.net>` and `bar@hmcts.net`

Reply:
Was just _reusing_ what's there. With possible email configs